pkg/bootstrap/docker/openshift/ansible.go

```diff
@@ -0,0 +1,182 @@
+package openshift
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"text/template"
+
+	"github.com/golang/glog"
+
+	"github.com/openshift/origin/pkg/bootstrap/docker/errors"
+	"github.com/openshift/origin/pkg/bootstrap/docker/run"
+)
+
+const (
+	defaultAnsibleImageUser      = "root"
+	defaultOpenshiftAnsibleImage = "openshift-ansible"
+	deploymentTypeOrigin         = "origin"
+	deploymentTypeOCP            = "openshift-enterprise"
+	imageStreamCentos            = "centos7"
+)
+
+const defaultMetricsInventory = `
```

Comment:
@ewolinetz  @sdodson Please review for correctness.  Looking to make the inventory as minimalistic as possible.  Note I added `openshift={master:{ha: false}}` otherwise the handler to bounce the the openshift service would trigger and openshift cluster is not running with a service that can be restarted?  Alternate suggestions?